pkg/kapis/gitops/v1alpha1/argocd/handler.go

```diff
@@ -23,12 +23,9 @@ import (
 	"k8s.io/apiserver/pkg/authentication/user"
```

Comment:
`argocd` folder was created for argo but why there does not exist a folder `fluxcd`?

Reply:
I extract some public APIs like CRUD on Applications outside the `argocd` folder and remain the Argo Exclusive APIs in the `argocd` folder. And I'll add `fluxcd` folder in the future.

<img width="328" alt="image" src="https://user-images.githubusercontent.com/53958238/190556679-3660ea5e-42e7-4657-904f-943c407d5c36.png">